packages/squiggle-lang/__tests__/Distributions/Symbolic_test.res

```diff
@@ -0,0 +1,177 @@
+open Jest
+open Expect
+
+let fnImage = (theFn, inps) => Js.Array.map(theFn, inps)
+
+let env: DistributionOperation.env = {
+  sampleCount: 100,
+  xyPointLength: 100,
+}
+
+let mkNormal = (mean, stdev) => GenericDist_Types.Symbolic(#Normal({mean: mean, stdev: stdev}))
+let {toFloat, toDist, toString, toError, fmap} = module(DistributionOperation.Output)
+let run = DistributionOperation.run(~env)
+let outputMap = fmap(~env)
+let toExtFloat: option<float> => float = E.O.toExt(
+  "Should be impossible to reach (This error is in test file)",
+)
+let toExtDist: option<GenericDist_Types.genericDist> => GenericDist_Types.genericDist = E.O.toExt(
+  "Should be impossible to reach (This error is in a test file)", 
+)
+let unpackFloat = x => x -> toFloat -> toExtFloat
+let unpackDist = y => y -> toDist -> toExtDist
+
+describe("(Symbolic) normalize", () => {
+  testAll("has no impact on normal distributions", list{-1e8, -16.0, -1e-2, 0.0, 1e-4, 32.0, 1e16}, mean => {
```

Comment:
Not sure if we need so many tests for this, the functionality is tiny here.

Reply:
i'll take some of the cases out of the list in the next commit.